src/MovieList.elm

```diff
@@ -959,6 +959,12 @@ movies =
       , genres = [ "comedy", "adventure" ]
       , watched = Unwatched
       }
+    , { title = "Donnie Darko"
+      , url = "http://www.imdb.com/title/tt0246578/"
+      , img = "donnie-darko.jpg"
+      , year = 2001
+      , runtime = 113
+      , genres = [ "drama", "sci-fi", "thriller" ]
```

Comment:
Add a `, watched = Unwatched` line. Build fails becuase that's missing.

Reply:
woops, messed up during merge conflict